pkg/runtime/kubernetes/environment_create.go

```diff
@@ -209,6 +312,36 @@ func (p generalProvisioner) EnvironmentCreate(ctx context.Context,
 		return nil, errors.Wrap(err, "failed to create pod")
 	}
 
+	servicePorts := []v1.ServicePort{
+		{
+			Name: "ssh",
+			Port: 2222,
```

Comment:
Could we make the  port number a const to avoid magic?

Reply:
Yes